File: src/main/java/org/project/model/response/StaffExperienceResponse.java
package org.project.model.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class StaffExperienceResponse {
    private Long id;
    private Integer year;
    private String department;
    private String position;
    private String hospital;
    private String result;
}

File: src/main/resources/templates/shared/assets/js/model/response/StaffExperienceResponse.js
export class StaffExperienceResponse {
    constructor(id, year, department, position, hospital, result) {
        this.id = id;
        this.year = year;
        this.department = department;
        this.position = position;
        this.hospital = hospital;
        this.result = result;
    }

    static fromJson(json) {
        return new StaffExperienceResponse(
            json.id,
            json.year,
            json.department,
            json.position,
            json.hospital,
            json.result
        );
    }

    static fromJsonArray(jsonArray) {
        return jsonArray.map(json => StaffExperienceResponse.fromJson(json));
    }
}
